### PR TITLE
Update data-classification-overview.md

### PR DESCRIPTION
**Pull request by Bhavanesh**

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/data-classification-overview.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ search.appverid:
 description: "The data classification dashboard gives you visibility into how much sensitive data has been found and classified in your organization."
 ---
 
-# Data classification overview (preview)
+# Know your data - Data classification overview (preview)
 
 As a Microsoft 365 administrator or compliance administrator, you can evaluate and then tag content in your organization in order to control where it goes, protect it no matter where it is and to ensure that it is preserved and deleted according your your organizations needs. You do this through the application of [sensitivity labels](sensitivity-labels.md), [retention labels](labels.md), and sensitive information type classification. There are various ways to do the discovery, evaluation and tagging, but the end result is that you may have very large numbers of documents and emails that are tagged and classified with one or both of these labels. After you apply  your retention labels and sensitivity labels, you’ll want to see how the labels are being used across your tenant and what is being done with those items. The data classification page provides visibility into that body of content, specifically:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-overview.md:23]:
```suggestion
As a Microsoft 365 administrator or compliance administrator, you can evaluate and then tag content in your organization in order to control where it goes, protect it no matter where it is and to ensure that it is preserved and deleted according your your organization's needs. You do this through the application of [sensitivity labels](sensitivity-labels.md), [retention labels](labels.md), and sensitive information type classification. There are various ways to do the discovery, evaluation and tagging, but the end result is that you may have very large numbers of documents and emails that are tagged and classified with one or both of these labels. After you apply  your retention labels and sensitivity labels, you’ll want to see how the labels are being used across your tenant and what is being done with those items. The data classification page provides visibility into that body of content, specifically:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - please also add apostrophe to Line 23 as indicated.

**Comment by mypil** [comment]
@Bhavanesh - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@Bhavanesh - We would like to follow up on this. Can you please merge the commit on the copy/edit review done by JohanFreelancer9?

Thank you.

**Comment by Bhavanesh** [comment]
Chris - did you accept this?


Sent from my phone.
Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by Bhavanesh** [comment]
@Meryljoyce Ypil<mailto:ypilmeryljoyce@gmail.com>

Hi Bhavy - I don't see this on my report of outstanding PR and GH issues so I have missed it.

https://msit.powerbi.com/groups/me/reports/4db881b0-ba34-401a-b1da-a2173baad7ac/ReportSection?ctid=72f988bf-86f1-41af-91ab-2d7cd011db47

I'll look more deeply into GH to find it. Sorry for the delay

Chris

**Comment by chrfox** [comment]
Found it.  I’ll get it reviewed and get back to you.

@Bhavanesh Rengarajan<mailto:Bhavanesh.Rengarajan@microsoft.com> it would help get the notification through faster if you @ mention me in the PR.

Thanks,

Chris

**Comment by chrfox** [comment]
Merged, should be live sometime today

From: Christopher J Fox (OFFICE)
Sent: Thursday, February 20, 2020 10:28 AM
To: 'MicrosoftDocs/microsoft-365-docs' <reply@reply.github.com>; MicrosoftDocs/microsoft-365-docs <microsoft-365-docs@noreply.github.com>; Bhavanesh Rengarajan <Bhavanesh.Rengarajan@microsoft.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: RE: [MicrosoftDocs/microsoft-365-docs] Update data-classification-overview.md (#1217)

Found it.  I’ll get it reviewed and get back to you.

@Bhavanesh Rengarajan<mailto:Bhavanesh.Rengarajan@microsoft.com> it would help get the notification through faster if you @ mention me in the PR.

Thanks,

Chris

**Comment by chrfox** [comment]
Thanks a lot @Christopher<mailto:chrfox@exchange.microsoft.com>.

Bhavy

From: Christopher J Fox (OFFICE) <chrfox@exchange.microsoft.com>
Sent: Thursday, February 20, 2020 10:37 AM
To: MicrosoftDocs/microsoft-365-docs <reply@reply.github.com>; MicrosoftDocs/microsoft-365-docs <microsoft-365-docs@noreply.github.com>; Bhavanesh Rengarajan <Bhavanesh.Rengarajan@microsoft.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: RE: [MicrosoftDocs/microsoft-365-docs] Update data-classification-overview.md (#1217)

Merged, should be live sometime today

From: Christopher J Fox (OFFICE)
Sent: Thursday, February 20, 2020 10:28 AM
To: 'MicrosoftDocs/microsoft-365-docs' <reply@reply.github.com<mailto:reply@reply.github.com>>; MicrosoftDocs/microsoft-365-docs <microsoft-365-docs@noreply.github.com<mailto:microsoft-365-docs@noreply.github.com>>; Bhavanesh Rengarajan <Bhavanesh.Rengarajan@microsoft.com<mailto:Bhavanesh.Rengarajan@microsoft.com>>
Cc: Subscribed <subscribed@noreply.github.com<mailto:subscribed@noreply.github.com>>
Subject: RE: [MicrosoftDocs/microsoft-365-docs] Update data-classification-overview.md (#1217)

Found it.  I'll get it reviewed and get back to you.

@Bhavanesh Rengarajan<mailto:Bhavanesh.Rengarajan@microsoft.com> it would help get the notification through faster if you @ mention me in the PR.

Thanks,

Chris

**Pull request merged**